collect-logs-devstack.yaml

```diff
@@ -46,7 +46,8 @@
       shell: |
         for u in $(sudo systemctl list-unit-files | grep devstack | awk '{print $1}'); do
             name=$(echo $u | sed 's/devstack@//' | sed 's/\.service//')
-            sudo journalctl -o short-precise --unit $u > {{ devstack_dir.stack }}/logs/$name.txt
+            timestamp=$(systemctl show -p ActiveEnterTimestamp $u)
```

Comment:
why do we need to use --since when getting logs? does it not retrieve all the logs without it ?